lib/hanami/validations/form.rb

```diff
@@ -12,57 +10,65 @@ module Validations
     # @since 0.6.0
     #
     # @example
-    #   require 'hanami/validations/form'
+    #   require "hanami/validations/form"
     #
     #   class Signup
     #     include Hanami::Validations::Form
     #
     #     validations do
-    #       required(:name).filled(:str?)
-    #       optional(:location).filled(:str?)
+    #       required(:name).value(:string)
+    #       optional(:location).value(:string)
```

Comment:
Same deal here, these should be `filled(:string)` if you want the example to express the same behaviour as before

Reply:
Fixed by https://github.com/hanami/validations/pull/180/commits/eef2c6da00000446c045fe94b951f27466a6911c